README.md

```diff
@@ -110,23 +98,94 @@ This project is licensed under the AGPL-3.0 License - see the [LICENSE.md](LICEN
-` { "All": 0, "Chapter": 1, "Slide": 2, "Hotspot": 3, "AnswerOnAir": 4, } ); `
+#### Configuration Structure
+
+```js
+//Default configuration
+"navigation" = {};
+//Plugin params
+"navigation" = {
+  expandOnFirstPlay?: boolean, // optional
+  forceChaptersThumb?: boolean, // optional
+  expandMode?: string, // optional
+  userRole?: string, // optional
+  itemsOrder?: object< string, number > // optional
+}
+```
 
-## Troubleshooting
+##
+
+> ### config.expandOnFirstPlay
+>
+> ##### Type: `boolean`
+>
+> ##### Default: `true`
+>
+> ##### Description: if plugin should automatically opens on first play.
+>
+
+##
+
+> ### config.forceChaptersThumb
+>
+> ##### Type: `boolean`
+>
+> ##### Default: `false`
+>
+> ##### Description: force to use chapters thumbnails.
+>
+
+##
+
+> ### config.expandMode
+>
+> ##### Type: `string`
+>
+> ##### Default: `alongside`
+>
+> ##### Description: expand mode of side panel (‘alongside', ‘hidden’, 'over’, default 'alongside').
+>
+
+##
+
+> ### config.userRole
+>
+> ##### Type: `string`
+>
+> ##### Default: `anonymousRole`
+>
+> ##### Description: use session userId as identificator of user (anonymousRole|unmoderatedAdminRole, default "anonymousRole").
+>
+
+##
+
+> ### config.itemsOrder
+>
+> ##### Type: `object< string, number >`
+>
+> ##### Default: `{ "All": 0, "Chapter": 1, "Slide": 2, "Hotspot": 3, "AnswerOnAir": 4, }`
+>
+> ##### Change tab order: `navigation = {...itemsOrder: { "Slide": 1, "All": 4, "AnswerOnAir": 3, "Chapter": 5,  "Hotspot": 2, }, ...}`
+>
+> ##### Description: define order of Tabs (min value at the left) and Items inside group (min value at the top). itemsOrder also uses as tabs filter (all items that not included in "itemsOrder" object will be filtered out) (default: { "All": 0, "Chapter": 1, "Slide": 2, "Hotspot": 3, "AnswerOnAir": 4, }).
+>
+
+## Additional flashvars
+"playkit-navigation":"Version" (check latest version of navigation plugin)
+
+<a name="dependencies"></a>
+## Dependencies
+
+Plugin dependencies:<br/>
+<a href="https://github.com/kaltura/playkit-js-kaltura-cuepoints">Cue Points</a><br/>
+<a href="https://github.com/kaltura/playkit-js-ui-managers">UI Managers</a>
 
 ### Dev env
-
-Node version: up to 14.18.1<br/>
+Node version: up to 14+<br/>
 If nvm installed: `nvm use` change version of current terminal to required.<br/>
```

Comment:
`nvm use` command requires `.nvmrc` file. Navigation plugin doesn't have such file, so OR remove the description for `nvm use` OR add missed `.nvmrc` file.